File: test/easy/RemoveArrayDuplicatesTest.php

<?php

use Src\Easy\RemoveArrayDuplicates;

describe('TestValue', function () {

    test('Testing', function ($candidate, $expected) {
        $instance = new RemoveArrayDuplicates;

        expect($instance->solve($candidate))->toBe($expected);

    })->with([
        [[1, 1, 2], 2],
        [[1, 2, 2, 3, 3], 3],
        [[1, 1, 2, 2, 3, 3, 4], 4],
    ]);

});

File: test/easyswift/RemoveArrayDuplicatesTest.swift
import Testing

@testable import easy

@Test(arguments: [
  ([1, 1, 2], 2),
  ([1, 2, 2, 3, 3], 3),
  ([1, 1, 2, 2, 3, 3, 4], 4),
])
func removeArrayDuplicatesTest(nums: [Int], outcome: Int) {
  var mutable = nums
  #expect(removeArrayDuplicates(&mutable) == outcome)
}
